apps/explorer/tests/transaction.spec.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { expect, test } from '@playwright/test';
+
+import { faucet, mint } from './utils/localnet';
+
+test('displays the transaction timestamp', async ({ page }) => {
+    const address = await faucet();
+    const tx = await mint(address);
+    const txid = tx.EffectsCert.certificate.transactionDigest;
+    const dateLike = /^[A-Za-z]{3}\b.+?(?=(AM|PM))/;
```

Comment:
I don't love this, we really shouldn't be dependent on the formatting at all (that can be done in unit tests for some `formatDate` function if we really want. Maybe just seeing if it's visible and has _some_ text would be fine?

Reply:
Yea, I don't like it much either. I'll update